package.json

```diff
@@ -26,6 +26,7 @@
     "google-protobuf": "^3.11.4",
     "immutable": "^4.0.0-rc.12",
     "rxjs": "^6.5.5",
+    "semver": "^5.7.1",
```

Comment:
Shouldn't this be a dev dependency?

Reply:
This one needs to be a dependency because it'll be used when the npm package gets installed.